src/pages/workspace/receiptPartners/InviteReceiptPartnerPolicyPage.tsx

```diff
@@ -0,0 +1,250 @@
+import {formatMemberForList, getHeaderMessage, getSearchValueForPhoneOrEmail, sortAlphabetically} from '@libs/OptionsListUtils';
+import type {MemberForList} from '@libs/OptionsListUtils';
+import {getPersonalDetailByEmail} from '@libs/PersonalDetailsUtils';
+import {isDeletedPolicyEmployee} from '@libs/PolicyUtils';
+import tokenizedSearch from '@libs/tokenizedSearch';
+import type {WorkspaceSplitNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+
+const MINIMUM_MEMBER_TO_SHOW_SEARCH = 8;
+
+type InviteReceiptPartnerPolicyPageProps = PlatformStackScreenProps<WorkspaceSplitNavigatorParamList, typeof SCREENS.WORKSPACE.RECEIPT_PARTNERS_INVITE>;
+
+function InviteReceiptPartnerPolicyPage({route}: InviteReceiptPartnerPolicyPageProps) {
+    const styles = useThemeStyles();
+    const {translate, localeCompare} = useLocalize();
+    const {isOffline} = useNetwork();
+    const [searchTerm, debouncedSearchTerm, setSearchTerm] = useDebouncedState('');
+    const [selectedOptions, setSelectedOptions] = useState<MemberForList[]>([]);
+    const [isInvitationSent, setIsInvitationSent] = useState(false);
+
+    const policyID = route.params?.policyID;
+    const policy = usePolicy(policyID);
+    const shouldShowSearchInput = policy?.employeeList && Object.keys(policy.employeeList).length >= MINIMUM_MEMBER_TO_SHOW_SEARCH;
+    const textInputLabel = shouldShowSearchInput ? translate('common.search') : undefined;
+
+    const workspaceMembers = useMemo(() => {
+        let membersList: MemberForList[] = [];
+        if (!policy?.employeeList) {
+            return membersList;
+        }
+
+        Object.entries(policy.employeeList).forEach(([email, policyEmployee]) => {
+            if (isDeletedPolicyEmployee(policyEmployee, isOffline)) {
+                return;
+            }
+
+            const personalDetail = getPersonalDetailByEmail(email);
+            if (personalDetail) {
+                const memberForList = formatMemberForList({
+                    text: personalDetail?.displayName ?? email,
+                    alternateText: email,
+                    login: email,
+                    accountID: personalDetail?.accountID,
+                    icons: [
+                        {
+                            source: personalDetail?.avatar ?? Expensicons.FallbackAvatar,
+                            name: formatPhoneNumber(email),
+                            type: CONST.ICON_TYPE_AVATAR,
+                            id: personalDetail?.accountID,
+                        },
+                    ],
+                    reportID: '',
+                    keyForList: email,
+                });
+
+                membersList.push({
+                    ...memberForList,
+                    isSelected: true,
```

Comment:
I don't get it, why are you doing this instead of passing `isSelected: true` to the `formatMemberForList`?